### PR TITLE
modify java.md, python.md, and configure-introduction.md

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
Signed-off-by: Liuxiaozhen12 <liuxiaozhen@pingcap.com>

### Any related PRs or issues?
https://github.com/tikv/website/pull/239
https://github.com/tikv/website/pull/233

### CODE REVIEW

**Comment by Liuxiaozhen12** [comment]
@marsishandsome @TomShawn  PTAL, thanks!

**Review by marsishandsome — APPROVED** [review]
LGTM

File: content/docs/5.1/deploy/configure/introduction.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ menu:
         weight: 2
 ---
 
-Though it's recommended to start with the default configurations when getting started with TiKV, TiKV features a large number of configuration parameters to tweak its behavior, which enables users to config the cluster to fit for the special application requirement.
+It's recommended to get started with TiKV with default configurations, however, TiKV features a large number of configuration parameters to tweak its behavior, allowing you to configure the cluster to suit the special application requirements.
```
Comment (TomShawn) [inline content/docs/5.1/deploy/configure/introduction.md:10]: It's --> It is 
Please spell it in full.

Comment (TomShawn) [inline content/docs/5.1/deploy/configure/introduction.md:10]:
```suggestion
It is recommended to get started with TiKV with default configurations. However, TiKV features a large number of configuration parameters to tweak its behavior, allowing you to configure the cluster to suit your special application requirements.
```

File: content/docs/5.1/deploy/configure/introduction.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ menu:
...
 
-The following topics describes how to config different TiKV components:
+The following guides provide you ways to configure different TiKV components:
```
Comment (TomShawn) [inline content/docs/5.1/deploy/configure/introduction.md:12]:
```suggestion
The following documents guide you on how to configure different TiKV components:
```

File: content/docs/5.1/develop/clients/java.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ menu:
         weight: 1
 ---
 
-This guide introduces how to interact with TiKV using [Java Client](https://github.com/tikv/client-java).
+This document guides you through how to use [Java Client](https://github.com/tikv/client-java) in TiKV.
```
Comment (TomShawn) [inline content/docs/5.1/develop/clients/java.md:10]:
```suggestion
This document guides you on how to use [Java Client](https://github.com/tikv/client-java) in TiKV.
```

File: content/docs/5.1/develop/clients/java.md [hunk 4]
```diff
@@ -73,9 +73,9 @@ for(Kvrpcpb.KvPair pair : list) {
 }
 ```
 
-These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete` and `batchScan`), which offer considerably reduced network overhead and can result in dramatic performance increases under certain workloads.
+These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`), which considerably reduce network overhead and increase performance under certain workloads.
```
Comment (TomShawn) [inline content/docs/5.1/develop/clients/java.md:76]:
```suggestion
These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`), which considerably reduces network overhead and increases performance under certain workloads.
```

File: content/docs/5.1/develop/clients/java.md [hunk 5]
```diff
@@ -73,9 +73,9 @@ for(Kvrpcpb.KvPair pair : list) {
...
 
-You can find all the functions that `RawKVClient` supports [here](https://github.com/tikv/client-java/blob/master/src/main/java/org/tikv/raw/RawKVClient.java).
+You can find all the functions `RawKVClient` supports [here](https://github.com/tikv/client-java/blob/master/src/main/java/org/tikv/raw/RawKVClient.java).
```
Comment (TomShawn) [inline content/docs/5.1/develop/clients/java.md:78]:
```suggestion
You can find all functions supported by `RawKVClient` [in this JAVA file](https://github.com/tikv/client-java/blob/master/src/main/java/org/tikv/raw/RawKVClient.java).
```

File: content/docs/5.1/develop/clients/python.md [hunk 6]
```diff
@@ -9,7 +9,7 @@ menu:
 
 TiKV client for python is built on top of [TiKV Client in Rust](https://github.com/tikv/client-rust) via CFFI and [PyO3 Python binding](https://github.com/PyO3/pyo3).
 
-This client is still in the stage of prove-of-concept and under heavy development. You can track development at [tikv/client-py](https://github.com/tikv/client-py/) repository.
+Python client is still in the stage of prove-of-concept and under heavy development. You can track development at [tikv/client-py](https://github.com/tikv/client-py/) repository.
```
Comment (TomShawn) [inline content/docs/5.1/develop/clients/python.md:12]:
```suggestion
The Python client is still in the stage of proof-of-concept and under development. You can track the development at [tikv/client-py](https://github.com/tikv/client-py/) repository.
```

**Comment by marsishandsome** [comment]
@Liuxiaozhen12 PTAL

File: content/docs/5.1/develop/clients/java.md (hunk 4 above)
Comment (Liuxiaozhen12) [reply content/docs/5.1/develop/clients/java.md:76]: Here I think `which` means `batch variants` and it is in plural form.

Comment (TomShawn) [reply content/docs/5.1/develop/clients/java.md:76]: Then please remove the comma before `which`.

File: content/docs/5.1/deploy/configure/introduction.md [hunk 7]
```diff
@@ -7,9 +7,9 @@ menu:
         weight: 2
 ---
 
-Though it's recommended to start with the default configurations when getting started with TiKV, TiKV features a large number of configuration parameters to tweak its behavior, which enables users to config the cluster to fit for the special application requirement.
+It is recommended to get started with TiKV with default configurations. However, TiKV features a large number of configuration parameters to tweak its behavior, allowing you to configure the cluster to suit your special application requirements.
```
Comment (TomShawn) [inline content/docs/5.1/deploy/configure/introduction.md:10]:
```suggestion
Although you are recommended to get started with TiKV using the default configuration, TiKV provides many configuration parameters to tweak its behavior, which allows you to configure the cluster to suit your special application requirements.
```

File: content/docs/5.1/deploy/configure/introduction.md [hunk 8]
```diff
@@ -7,9 +7,9 @@ menu:
...
 
-The following topics describes how to config different TiKV components:
+The following documents guide you on how to configure different TiKV components:
```
Comment (TomShawn) [inline content/docs/5.1/deploy/configure/introduction.md:12]:
```suggestion
The following list of documents guide you on how to configure different TiKV components:
```

File: content/docs/5.1/deploy/configure/introduction.md [hunk 9]
```diff
@@ -26,7 +26,7 @@ TiKV
 - [Region Merge](../region-merge): Tweak region merging.
```
Comment (TomShawn) [inline content/docs/5.1/deploy/configure/introduction.md:26]:
```suggestion
- [Region Merge](../region-merge): Tweak Region merging.
```

File: content/docs/5.1/develop/clients/java.md (hunk 4 above)
Comment (Liuxiaozhen12) [inline content/docs/5.1/develop/clients/java.md:76]:
```suggestion
These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`) which considerably reduce network overhead and increase performance under certain workloads.
```

Comment (Liuxiaozhen12) [reply content/docs/5.1/develop/clients/java.md:76]: Done. PTAL~

Comment (Liuxiaozhen12) [inline content/docs/5.1/develop/clients/java.md:76]:
```suggestion
These functions also have batch variants (`batchPut`, `batchGet`, `batchDelete`, and `batchScan`) which considerably reduce network overhead and increase performance under certain workloads.
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: content/docs/5.1/deploy/configure/introduction.md [hunk 10]
```diff
@@ -7,9 +7,9 @@ menu:
         weight: 2
 ---
 
-Though it's recommended to start with the default configurations when getting started with TiKV, TiKV features a large number of configuration parameters to tweak its behavior, which enables users to config the cluster to fit for the special application requirement.
+Although you are recommended to get started with TiKV using the default configuration, TiKV provides many configuration parameters to tweak its behavior, which allows you to configure the cluster to suit your special application requirements.
 
-The following topics describes how to config different TiKV components:
+The following list of documents guide you on how to configure different TiKV components:
```
Comment (TomShawn) [inline content/docs/5.1/deploy/configure/introduction.md:12]:
```suggestion
The following lists of documents guide you on how to configure different TiKV components:
```

**Pull request merged**